java/spring-boot/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.5.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>benchmark.spring-boot</groupId>
+	<artifactId>benchmark</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>benchmark</name>
+	<description>Demo project for Spring Boot</description>
+
+	<properties>
+		<java.version>1.8</java.version>
```

Comment:
Why this has to be specified ?

Reply:
It is an extra insurance the build will be in the intended version. In the case of travis ci already ensured by the Dockerfile but on local dev it is not. So, not absolutely mandatory but a good practice.

Reply:
ok

Reply:
If it is a good practice, leave it :stuck_out_tongue: